### ISSUE TITLE
Notebook showing how to use GeoTrellis

### ISSUE THREAD

**Issue by andypetrella** [andypetrella/spark-notebook#74]
that would be a great show up 
/cc @lossyrob ^^

**Comment by lossyrob** [comment]
@andypetrella agreed! Haven't used Scala Notebook yet but want to get ramped up. Let's talk about it\hack on it at FOSSDEM.

**Comment by andypetrella** [comment]
done deal!

However, I'll be at fosdem Sunday only.
I'll be giving a Devoxx4Kids to 50 kids the whole Saturday and the evening
I've a dinner with wife and kid at school...
Anyway, this will give us some room for hacking :-D

Until when are you in Belgium?

**Comment by lossyrob** [comment]
I'm there until Tuesday morning. No plans on what to do on Monday, but I
wanted to get a sense of what Brussels was all about.
